src/renderer/src/view_models/XViewModel.ts

```diff
@@ -1405,7 +1405,7 @@ Please wait while I index all the messages from each conversation...`;
             if (!success) {
                 await this.error(AutomationErrorType.x_runJob_indexMessages_Timeout, {
                     exception: (error as Error).toString(),
-                }, true);
```

Comment:
looking at the signature of the `BaseViewModel` `error` method:
`async error(automationErrorType: AutomationErrorType, errorReportData: any = null, sensitiveContextData: any = null, allowContinue: boolean = false)`, I'm wondering if the intent here was that this bool be the `allowContinue` field?

Reply:
Nope I don't think that was the intent. Because looking at the code, it loads the DM conversation URL and waits for messages to appear, and so if the message never appear, it shouldn't allow continue.

Reply:
ahh ok, thanks